content-updates.md

```diff
@@ -220,3 +222,42 @@ Example:
   description="Learn Django, a python-based web framework for building backends, APIs, multi-page applications, and more!"
 />
 ```
+
+## Plawright Tests
+
+### Installing Plawright
+
+Here are 2 helpful links to get you started with Plawright
+[Plawright Getting Started ](https://playwright.dev/docs/intro) Or [Getting Started with Playwright and VS Code](https://www.youtube.com/watch?v=Xz6lhEzgI5I). 
+The second link is a video tutorial that will guide you through the process of setting up Plawright with VS Code, writing tests, and running them using the IDE.
+
+### How to run the tests using the terminal
+
+This will run all the tests in the tests folder
+
+```bash
+npx playwright test
```

Comment:
we should probably change the test script in package.json to run this command instead

Reply:
Can you explain what you mean pls.
Are you referring to changing line 48 in `package.json`?
<img width="409" alt="image" src="https://github.com/user-attachments/assets/80e0a235-811c-4660-b9e1-316982dd97ea">